src/routes/v1/organisations/get-by-id.js

```diff
@@ -0,0 +1,31 @@
+import Boom from '@hapi/boom'
+import { StatusCodes } from 'http-status-codes'
+
+/** @typedef {import('#repositories/organisations/port.js').OrganisationsRepository} OrganisationsRepository */
+
+export const organisationsGetByIdPath = '/v1/organisations/{orgId}'
```

Comment:
Is this to be used for searching by OrgID or more typical use cases like returning a single organisation? If it's the latter, then it would be much safer to use the `_id` on the `epr_organisations` entity